Makefile

```diff
@@ -144,11 +154,48 @@ test-e2e-olm-ci:
 # Build and vendor tarets
 #---------------------------------------------------------
 
-# Vendor target: "go mod vendor" resets the main module's vendor directory to include all packages needed to build and 
-# test all of the module's packages based on the state of the go.mod files and Go source code.
+.PHONY: build-image
+## Build: using operator-sdk to build a new image
+build-image:
+	$(Q)GO111MODULE=on operator-sdk build "$(OPERATOR_IMAGE):$(OPERATOR_TAG_LONG)"
+
+## Vendor: "go mod vendor" resets the vendor folder to what's defined in go.mod
 ./vendor: go.mod go.sum
 	$(Q)GOCACHE=$(shell pwd)/out/gocache GO111MODULE=on go mod vendor ${V_FLAG}
 
+## Generate CSV: using oeprator-sdk generate cluster-service-version for current operator version
+generate-csv:
+	operator-sdk olm-catalog gen-csv --csv-version=$(OPERATOR_VERSION) --verbose
+
+generate-olm:
+	operator-courier --verbose flatten $(MANIFESTS_DIR) $(MANIFESTS_TMP)
+	cp -vf deploy/crds/*_crd.yaml $(MANIFESTS_TMP)
+
+#---------------------------------------------------------
+# Deploy
+#---------------------------------------------------------
+
+## Prepare-CSV: using a temporary location copy all operator CRDs and metadata to generate a CSV.
+prepare-csv: build-image
+	$(eval ICON_BASE64_DATA := $(shell cat ./assets/icon/red-hat-logo.png | base64))
```

Comment:
Probably it's not a good idea to use Red Hat old logo here.
For now, we can use some placeholder image.
Check this site to other product logos usage: https://operatorhub.io/
BTW, here is the new Red Hat logo details:
https://www.redhat.com/en/about/brand/standards/logo

Reply:
Good point. I just wanted a temporary logo, and now I'm picking up one from the link you shared. Please consider.